needed because, vim yields:
syntastic: error: checker rust/rustc returned abnormal status 101       
and thus does not show the syntax errors, like it should!

https://github.com/rust-lang/rfcs/issues/734

diff --git a/syntax_checkers/rust/rustc.vim b/syntax_checkers/rust/rustc.vim
index 5d19608..ca62308 100644
--- a/syntax_checkers/rust/rustc.vim
+++ b/syntax_checkers/rust/rustc.vim
@@ -24,7 +24,8 @@ function! SyntaxCheckers_rust_rustc_GetLocList() dict
 
     return SyntasticMake({
         \ 'makeprg': makeprg,
-        \ 'errorformat': errorformat })
+        \ 'errorformat': errorformat,
+        \ 'returns': [0,101] }) " 0 is normal, -1 is a result of C-c (unusable with syntactic still); 101 is due to compilation error! (usable)
 endfunction
 
 call g:SyntasticRegistry.CreateAndRegisterChecker({
